packages/mask/src/index.js

```diff
@@ -164,6 +164,8 @@ export function buildUp(template, input) {
 }
 
 function formatMoney(input, delimeter = '.', thousands) {
+    if (!input.replace(/\D+/g, '')) return '9'
```

Comment:
Shouldn't this be a regex test and not a replace?

Reply:
@ekwoka Yes, it is possible to use the both

Reply:
I don't mean whats possible, I mean what's the correct tool for the job?

Wouldn't

```js
if (!/^\D+$/.test(input)) return '9'
```

Be more explicit to what you're actually doing?